Source/Model/ParserZ3.cs

```diff
@@ -281,6 +281,9 @@ internal override void Run()
           }
           else
           {
+            if (funName.StartsWith("aux$$")) {
```

Comment:
Does the name `aux$$` occur anywhere else in the code? Can we use a string constant to refer to it?

Reply:
Yes, it occurs in a small number of other places. We probably should create a string constant for it. I'll do that in a future refactoring.